feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go

```diff
@@ -95,3 +96,56 @@ func TestPrefixSet(t *testing.T) {
 		}
 	}
 }
+
+func TestPrefixSetWithCLIConfig(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+
+	prefixSetConfig := `
+	ip prefix-list TAG_3_IPV4
+	  seq 10 permit 10.240.31.48/28
+   	  seq 20 permit 10.244.187.32/28
+   	  seq 30 permit 173.36.128.0/20
+   	  seq 40 permit 173.37.128.0/20
+	  seq 50 permit 173.38.128.0/20
+	  seq 60 permit 173.39.128.0/20
+	  seq 70 permit 173.40.128.0/20
+	  seq 80 permit 173.41.128.0/20
+	  seq 90 permit 173.42.128.0/20
+	  seq 100 permit 173.43.128.0/20
+	`
+	helpers.GnmiCLIConfig(t, dut, prefixSetConfig)
```

Comment:
this is a vendor-specific cli/gnmi request, but not marked as such.

Reply:
added switch case for vendor specific config